packages/editor/src/components/post-featured-image/index.js

```diff
@@ -95,6 +95,27 @@ function PostFeaturedImage( {
 		<PostFeaturedImageCheck>
 			{ noticeUI }
 			<div className="editor-post-featured-image">
+				{ media && (
+					<div
+						id={ `editor-post-featured-image-${ featuredImageId }-describedby` }
+						className="hidden"
+					>
+						{ media.alt_text &&
+							sprintf(
+								// Translators: %s: The selected image alt text.
+								__( 'Current image: %s' ),
+								media.alt_text
+							) }
+						{ ! media.alt_text &&
+							sprintf(
+								// Translators: %s: The selected image slug.
```

Comment:
This comment should not be updated, as it's not the slug any longer 🙂

Reply:
Fixed (changed `slug` to `filename`) :+1: